File: 프로그래머스/lv0/120845. 주사위의 개수/주사위의 개수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> box, int n)
{
    int answer;

    int a = box[0] / n;
    int b = box[1] / n;
    int c = a * b;
    answer = c * (box[2] / n);

    return answer;
}

File: 프로그래머스/0/120845. 주사위의 개수/주사위의 개수.java
class Solution {
    public int solution(int[] box, int n) {
        int answer = 1;
        
        int row = box[0];
        int col = box[1];
        int height = box[2];
        
        answer *= row / n;
        answer *= col / n;
        answer *= height / n;
        
        return answer;
    }
}